x-pack/plugins/security_solution_serverless/server/cloud_security/cloud_security_metering.ts

```diff
@@ -21,6 +21,15 @@ export const cloudSecurityMetringCallback = async ({
   lastSuccessfulReport,
   config,
 }: MeteringCallbackInput): Promise<MeteringCallBackResponse> => {
```

Comment:
Following [FP alerts](https://elastic.slack.com/archives/C056S4ZJW3W/p1718630716976019) we had, I found it more effective to skip cloud metering earlier.